### PR TITLE
feat(icon): add pdf and ppt icons

### PR DESCRIPTION
**Pull request by michael-iden**
**This change:** (check at least one)

- [X] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [ ] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [X] No

**Is the:** (complete all)

- [X] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [X] Test suite(s) passing?
- [X] Code coverage maximal?
- [X] Changeset added?

**What does this change address?**
This adds two new svg icons to denote export file formats

**How does this change work?**
Adds the icons to the assets directory and json registry

### CODE REVIEW

File: packages/pharos/assets/icons/pdf.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Niznikr) [inline packages/pharos/assets/icons/pdf.svg:1]: You can remove the outer `<svg>` as the component wraps that around these

File: packages/pharos/assets/icons/pdf.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M15.4142 4H4V20H20V8.58579L15.4142 4ZM6 18V6H14.5858L18 9.41421V18H6Z" fill="black"/>
```
Comment (Niznikr) [inline packages/pharos/assets/icons/pdf.svg:3]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M15.4142 4H4V20H20V8.58579L15.4142 4ZM6 18V6H14.5858L18 9.41421V18H6Z"/>
```
We avoid fills to allow folks to pick their own colors when needed.

File: packages/pharos/tokens/assets/icons.json [hunk 3]
```diff
@@ -55,6 +55,8 @@
       "menu": { "value": "assets/icons/menu.svg" },
       "move": { "value": "assets/icons/move.svg" },
       "new": { "value": "assets/icons/new.svg" },
+      "pdf": { "value": "assets/icons/pdf.svg" },
+      "ppt": { "value": "assets/icons/ppt.svg" },
```
Comment (daneah) [inline packages/pharos/tokens/assets/icons.json:59]: Should these be named something like `filetype-ppt` to (1) have them sort together and (2) reduce any ambiguity about what some might mean?

**Review by daneah — APPROVED** [review]

File: .changeset/early-bears-report.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos': minor
+---
+
+feat(icon): add pdf and ppt icons
```
Comment (Niznikr) [inline .changeset/early-bears-report.md:5]:
```suggestion
add pdf and ppt icons
```

File: packages/pharos/assets/icons/filetype-pdf.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M15.4142 4H4V20H20V8.58579L15.4142 4ZM6 18V6H14.5858L18 9.41421V18H6Z" fill="black"/>
```
Comment (Niznikr) [inline packages/pharos/assets/icons/filetype-pdf.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M15.4142 4H4V20H20V8.58579L15.4142 4ZM6 18V6H14.5858L18 9.41421V18H6Z"/>
```

File: packages/pharos/assets/icons/filetype-ppt.svg [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<path fill-rule="evenodd" clip-rule="evenodd" d="M9 16V8H11.5879C12.4352 8 13.0385 8.03443 13.3481 8.1212C13.8229 8.25174 14.2174 8.53742 14.5276 8.96169C14.8516 9.40136 15 9.95453 15 10.5922C15 11.0814 14.9154 11.5177 14.7284 11.8856C14.5517 12.2334 14.3205 12.5179 14.0319 12.7288C13.7637 12.9237 13.4812 13.0594 13.1863 13.1267L13.1803 13.1281C12.8121 13.2045 12.2994 13.2394 11.6546 13.2394H11.0604V16H9ZM12.7927 10.987C12.8595 10.8861 12.8967 10.7656 12.8967 10.6122C12.8967 10.4176 12.8443 10.2833 12.7567 10.1818C12.6579 10.0672 12.5346 9.99463 12.3733 9.96413L12.3708 9.96365C12.2373 9.93725 11.9295 9.91963 11.4165 9.91963H11.0604V11.3148H11.507C11.7788 11.3148 11.9992 11.3053 12.1706 11.2877C12.3475 11.2695 12.4519 11.2443 12.5035 11.2232C12.6339 11.17 12.7251 11.0922 12.7902 10.9909L12.7927 10.987Z" />
+<path fill-rule="evenodd" clip-rule="evenodd" d="M15.4142 4H4V20H20V8.58579L15.4142 4ZM6 18V6H14.5858L18 9.41421V18H6Z" fill="black"/>
```
Comment (Niznikr) [inline packages/pharos/assets/icons/filetype-ppt.svg:2]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M15.4142 4H4V20H20V8.58579L15.4142 4ZM6 18V6H14.5858L18 9.41421V18H6Z"/>
```

**Review by Niznikr — APPROVED** [review]

**Review by jialin-he — APPROVED** [review]

**Pull request merged**